winpr/include/winpr/winpr.h

```diff
@@ -61,6 +61,20 @@
 #define WINPR_NORETURN(obj) obj
 #endif
 
+#ifdef BUILD_TESTING
```

Comment:
Not sure that everyone will realize that building tests also make local symbols exportable...

Reply:
Perhaps we must find a way to have some FreeRDP internal unitary tests that don't rely on the public API...

Reply:
@hardening that is the reason we export all symbols with debug builds. otherwise we can not access most of the functions we test (maybe we can change the windows build to default to export without a macro?)

Reply:
@hardening ok, did update the code to now only use `EXPORT_ALL_SYMBOLS` instead of `BUILD_TESTING`. this way it should be more understandable